2.6.0/Dockerfile

```diff
@@ -47,6 +45,11 @@ ADD yarn-site.xml $HADOOP_CONF_DIR/yarn-site.xml
 ADD mapred-site.xml $HADOOP_CONF_DIR/mapred-site.xml
 ADD hdfs-site.xml $HADOOP_CONF_DIR/hdfs-site.xml
 
+# Creating symlink for HADOOP configuration files
+RUN mkdir /data && \
```

Comment:
Why put the `VOLUME` instructions after the format, which forces to do a `mkdir` ?

Is it because of docker caching? If so, please elaborate on the tests you've made and/or what drove you to do this change.